sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -236,13 +236,14 @@ ALTER TABLE `procedure_answers` ADD `procedure_code` VARCHAR(31) DEFAULT NULL;
 
 #IfNotRow users username oe-system
 INSERT INTO `users`(`username`,`password`,`lname`,`authorized`,`active`) VALUES ('oe-system','NoLogin','System Operation User',0,0);
-INSERT INTO `gacl_aro`(`id`, `section_value`, `value`, `order_value`, `name`, `hidden`)
-    SELECT max(`id`)+1,'users','oe-system',10,'System Operation User', 0 FROM `gacl_aro`;
+INSERT INTO `gacl_aro`(`id`, `section_value`, `value`, `order_value`, `name`, `hidden`) VALUES
+    ((SELECT max(`id`)+1),'users','oe-system',10,'System Operation User', 0 FROM `gacl_aro`);
```

Comment:
is above fixing something?
if so, then prob need in sql/official_additional_users.sql
if not, then would revert so is same as sql/official_additional_users.sql